compiler/resolution/wrappers.cpp

```diff
@@ -1678,7 +1824,8 @@ static void handleInIntents(FnSymbol* fn,
     Symbol* actualSym  = info.actuals.v[j];
 
     if (formalRequiresTemp(formal, fn) &&
-        shouldAddFormalTempAtCallSite(formal, fn)) {
+        shouldAddFormalTempAtCallSite(formal, fn) &&
+        actualSym->hasFlag(FLAG_DEFAULT_ACTUAL) == false) {
```

Comment:
It'd be nice to have a comment here saying something like "The result of a default argument for an 'in' intent is already owned and doesn't need to be copied"